plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1743,8 +1743,8 @@ dialog.about.message=<html><font style="font-size: 23px; font-weight:">\
   Full version: <b>{1}</b><br>\
   Current recommended major version: <b>{2}</b><br><br>\
   Workspace element format version: <b>{3}</b><br><br>\
-  OS architecture detected: <b>{4}-bit OS</b><br>\
-  Bundled JVM architecture: <b>{5}-bit JVM</b><br><br>\
+  OS architecture detected: <b>{4}-bit OS ({5})</b><br>\
+  Bundled JVM architecture: <b>{6}-bit JVM ({7})</b><br><br>\
```

Comment:
Showing arch next to JVM is confusing given you show OS arch here and not JVM arch